capistrano-antistatique.gemspec

```diff
@@ -24,7 +24,6 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'capistrano', '~> 3.5'
   spec.add_dependency 'capistrano-composer', '~> 0.0.6'
-  spec.add_dependency 'slackistrano', '~> 3.1'
```

Comment:
I suppose this break BC but slackistrano should be optional

Reply:
We could keep the gem dependency as that doesn't have so much effect. But if you want to remove it, why not.